samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp

```diff
@@ -323,11 +323,11 @@ document.links[newindex].onclick();
 				<h:outputText value="#{question.roundedMaxPointsToDisplay} #{deliveryMessages.pt}" rendered="#{delivery.actionString=='reviewAssessment'}"/>
 				<%-- DELIVER ASSESSMENT --%>
 				<h:outputText value="#{question.roundedMaxPoints}" rendered="#{delivery.settings.displayScoreDuringAssessments != '2' && question.itemData.scoreDisplayFlag && delivery.actionString!='reviewAssessment'}"  >
-					<f:convertNumber maxFractionDigits="2"/>
+					<f:convertNumber maxFractionDigits="2" groupingUsed="false"/>
 				</h:outputText>
-				<h:outputText value="#{deliveryMessages.pt}" rendered="#{delivery.settings.displayScoreDuringAssessments != '2' && question.itemData.scoreDisplayFlag && delivery.actionString!='reviewAssessment'}"  />
+				<h:outputText value=" #{deliveryMessages.pt}" rendered="#{delivery.settings.displayScoreDuringAssessments != '2' && question.itemData.scoreDisplayFlag && delivery.actionString!='reviewAssessment'}"  />
```

Comment:
To reviewers: this fixes the display issue: "1000Points" will now appear as "1000 Points"